addon/ng2/commands/serve.ts

```diff
@@ -0,0 +1,129 @@
+const assign      = require('lodash/assign');
+const Command     = require('ember-cli/lib/models/command');
+const Promise     = require('ember-cli/lib/ext/promise');
+const SilentError = require('silent-error');
+const PortFinder  = require('portfinder');
+const win         = require('ember-cli/lib/utilities/windows-admin');
+const EOL         = require('os').EOL;
+
+
+PortFinder.basePort = 49152;
+
+const getPort = Promise.denodeify(PortFinder.getPort);
+const defaultPort = process.env.PORT || 4200;
+
+export interface ServeTaskOptions {
+  port?: number;
+  host?: string;
+  proxy?: string;
+  insecureProxy?: boolean;
+  watcher?: string;
+  liveReload?: boolean;
+  liveReloadHost?: string;
+  liveReloadPort?: number;
+  liveReloadBaseUrl?: string;
+  liveReloadLiveCss?: boolean;
+  environment?: string;
+  outputPath?: string;
+  ssl?: boolean;
+  sslKey?: string;
+  sslCert?: string;
+}
+
+module.exports = Command.extend({
+  name: 'serve',
+  description: 'Builds and serves your app, rebuilding on file changes.',
+  aliases: ['server', 's'],
+
+  availableOptions: [
+    { name: 'port',                 type: Number,  default: defaultPort,   aliases: ['p'] },
+    { name: 'host',                 type: String,                          aliases: ['H'],     description: 'Listens on all interfaces by default' },
+    { name: 'proxy',                type: String,                          aliases: ['pr', 'pxy'] },
+    { name: 'insecure-proxy',       type: Boolean, default: false,         aliases: ['inspr'], description: 'Set false to proxy self-signed SSL certificates' },
+    { name: 'watcher',              type: String,  default: 'events',      aliases: ['w'] },
+    { name: 'live-reload',          type: Boolean, default: true,          aliases: ['lr'] },
```

Comment:
If you don't use those options, please remove them. I think webpack has its own HMR.